### ISSUE TITLE
[Question] Do blog entries show up in news feed?

### ISSUE THREAD

**Issue by SchrodingersGat** [inventree/inventree-website#70]
@matmair do blog entries get listed in the news feed?

**Comment by matmair** [comment]
@SchrodingersGat no. The idea is that news elements are short snippets that are shown in the webinterface and optimised for that. We can also regulate the number of news items showing up - ie not overloading the users.

**Comment by matmair** [comment]
I would suggest sending out news max twice a month, we can include a link list with that
